README.md

```diff
@@ -259,3 +260,19 @@ StarlingUI covers the following types of Grid:
 Check it out at: https://starlingui.netlify.app/pages/components/grid/grid.html
 
 ---
+
+### Slider
+
+Slider allows users to make selections from a range of values.
+
+StarlingUI covers the following types of Slider:
+
+- Standard Slider
+- Slider with Icon
+- Disabled Slider with Icon
+
+Check it out at: https://starlingui.netlify.app/pages/components/slider/slider.html
+
+---
```

Comment:
Good Work !!

Keeping up with the regular documentation

Suggestion : 

You can have a updation Tab for your component library  for the end users to get the latest updates you have done in your CL.

You can document your CL like that :)

Reply:
Thanks for the suggestion. It is a good idea. Will definitely keep it in the books for future implementation.